app/assets/stylesheets/_settings.scss

```diff
@@ -36,3 +40,5 @@ $search-filter-padding: $baseline-unit*4;
 $search-filter-label-color: $color-black;
 
 $color-link-blue: $color-blue-medium;
+$background-gradient-color: #f9fafa;
```

Comment:
I'm sure I could work it out by seeing how this is used but might be worth coming up with a more descriptive name than this?